execution_workflows/TAPAS/bin/make_tapas_ref.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import sys
+bed_path,gtf_path=sys.argv[1],sys.argv[2]
+def readGTF(gtf_path):
+ transcript2gene={}
+ gtf=open(gtf_path,"r")
+ for ln in gtf:
+  if not ln.startswith("#"):
+   attrList=ln.split("\t")[-1].split(";")
+   attrDict={}
+   for k in attrList:
+    p=k.strip().split(" ")
+    if len(p) == 2:
+     attrDict[p[0]]=p[1].strip('\"')
+   if "transcript_id" in attrDict:
+    transcript2gene[attrDict["transcript_id"]]=attrDict["gene_id"]
+ return transcript2gene
+transcript2gene=readGTF(gtf_path)
+
+tapas_ref_path="tapas_ref.txt"
+bed=open(bed_path,"r")
+outfile=open(tapas_ref_path,"w")
+for ln in bed:
+ ln=ln.split("\t")
+ transcript="NM_"+ln[3]
+ gene="NM_"+transcript2gene[ln[3]]
+ organized=[gene,transcript,ln[0],ln[5],ln[1],ln[2],ln[6],ln[7],ln[9],ln[10],ln[11]]
```

Comment:
I removed stripping the 'chr' prefix in 676bceceb9aea862f2bb3ebe768b5d2471c1f38a. With my test data I found this meant the reference and the coverage files no longer matched, so I got empty output. Fixing this got me proper output iwth the test data
![tapas-chr-strip-mismatch-2022-07-20-16-14-17](https://user-images.githubusercontent.com/49978382/180019016-3c369c48-0926-4893-9f06-e6478621c055.png)